compiler-rt/lib/mpk_untrusted/mpk_formatter.cpp

```diff
@@ -134,16 +134,21 @@ bool writeUniqueFile(std::set<std::shared_ptr<AllocSite>> &faultSet) {
 // allocations to disk/file.
 void flush_allocs() {
   auto handler = AllocSiteHandler::getOrInit();
-  if (handler->faultingAllocs().empty()) {
+  auto fault_set = handler->faultingAllocs();
+  if (fault_set.empty()) {
     REPORT("INFO : No faulting instructions to export, returning.\n");
     return;
   }
 
+  REPORT("INFO : Beginning faulting alloc flush.\n");
```

Comment:
message is not great, consider being explicit, like `serializing to disk ` or something like that

Reply:
Changed as requested.